.github/workflows/phpstan.yml

```diff
@@ -8,9 +8,18 @@ on:
 jobs:
   phpstan:
     name: Analyze
-    runs-on: [ubuntu-latest]
+    runs-on: ${{ matrix.operating-system }}
+    strategy:
+      matrix:
+        operating-system: [ubuntu-latest]
+        php-versions: ['7.4', '8.2', '8.3']
```

Comment:
Did you rule out 8.0 and 8.1? Even if they are LTS for the PHP team, these versions are not for a few OS's that provide them. The versions benefit from periodic updates until the end of LTS.

Reply:
I thought:
- we have to test for the minimum supported version (7.4)
- I wanted to start testing for the upcoming PHP version (8.3)
- it seems right to test for the most recent supported version, which is also the most restrictive (8.2)
- not to overload github with too many workflow I didn't add more